schemas/social-network-schema.gql

```diff
@@ -1,908 +0,0 @@
-define
```

Comment:
This is a repeat of a similar schema in the docs that is rapidly getting out of date. Deleted as it has no corresponding example package.